contracts/AccesorManager.sol

```diff
@@ -0,0 +1,75 @@
+/*
+ * Hubii Striim
+ *
+ * Compliant with the Hubii Striim specification v0.12.
+ *
+ * Copyright (C) 2017-2018 Hubii AS
+ */
+
+pragma solidity ^0.4.24;
+
+import {Ownable} from "./Ownable.sol";
+
+/**
+@title AccesorManager
+@notice A contract to control who can execute some specific actions
+*/
+contract AccesorManager is Ownable {
+    //
+    // Variables
+    // -----------------------------------------------------------------------------------------------------------------
+    address public operator;
+    mapping (address => bool) public signersMap;
+
+    //
+    // Events
+    // -----------------------------------------------------------------------------------------------------------------
+    event ChangeOperatorEvent(address oldOperator, address newOperator);
+    event RegisteredSignerEvent(address signer);
```

Comment:
I suggest `RegisterSignerEvent` to align on form of verb that defines name of event.